### ISSUE TITLE
Store contract command causing regen daemon hangs

### ISSUE THREAD

**Issue by novy4** [regen-network/testnets#900]
```
regen version v0.6.0
docker version  19.03.6      
cargo version cargo 1.50.0 (f04e7fab7 2021-02-04)
```

After running "sudo docker run" to build the contracts, I am executing:
`regen tx wasm store artifacts/cw20_base.wasm --chain-id aplikigo-1 --from <key_name> -y`
and the daemon just hangs, validator stoping to get/sign blocks. I tried to use more gas and fees without any luck.

The solution:
1. restart the cosmovisor daemon
2. change the user permissions for contracts built with docker
3. go in artifacts directory and execute the store command from that directory like:
 `regen tx wasm store cw20_base.wasm --chain-id aplikigo-1 --from my-key  --gas auto --fees 2000000utree -y`

**Comment by ama31337** [comment]
same here, confirm the bug
but in my case regen daemon stucked completely and no tx are executable, even regular trasnfers, so only daemon restart helped in my case

**Comment by ryanchristo** [comment]
CosmWasm is no longer included in the latest version of regen ledger (v5.0). We may consider adding cosmwasm when there is a version that is compatible with the latest cosmos sdk version but we encourage cosmwasm contract developers to leverage interchain accounts to interact with regen ledger features via IBC.
